pkg/controllers/user/resourcequota/resource_quota_cleanup.go

```diff
@@ -0,0 +1,55 @@
+package resourcequota
+
+import (
+	"github.com/rancher/types/apis/core/v1"
+	"github.com/rancher/types/apis/management.cattle.io/v3"
+	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+/*
+cleanupController watches resource quota resource, and cleanup those not created by Rancher
```

Comment:
Nit: suggested grammar changes because I had to read this more than once to grok it:
- drop the comma after resource
- s/cleanup/cleans up
- s/belong/belongs

Reply:
ok